### PR TITLE
a little more beginner friendly

### PR DESCRIPTION
**Pull request by System-x64**
#### Describe your PR, what does it fix/add?

- Added some of the options that I assume most people will add anyway (like media control keys)
- Added a new myTerminal variable to make it easy for newcomers to change their terminal(s)
- Made the hyprlinks a little more precise, now they connect directly to the tag instead of the page
- Made those basic options more visible (but still at its default state) which anyone skimming through the wiki would probably change (like inactive_opacity)
- Also added Mod + Return as a bind to open terminal (since most peep coming from X11wm have this as their only life saver to get out!
#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
- Yeah, i've added a cool looking HYPRLAND ascii art mostly because other linux configs have this type of layout aswell (although you can remove it if you want)

### CODE REVIEW

**Comment by vaxerski** [comment]
update autogen config too

**Comment by System-x64** [comment]
> update autogen config too

`autogenerated=1`?

**Comment by System-x64** [comment]
done!

**Comment by vaxerski** [comment]
no, idiot, the autogenerated config. it's in src/config/

**Comment by System-x64** [comment]
Oh ok, but could you be a little more specific next time, I mean `autogen config` could resemble a million different things

**Comment by vaxerski** [comment]
no

**Comment by System-x64** [comment]
ok...could u alteast accept this pull request now, I've changed that _autogen config_

**Comment by vaxerski** [comment]
I will accept it when I have time to re-review it, prolly in around an hour

**Comment by System-x64** [comment]
K

**Comment by System-x64** [comment]
Remove anything u don't like, cuz I've changed many stuff

**Comment by vaxerski** [comment]
you did not give me write perms to the PR.

**Comment by NotAShelf** [comment]
I can't submit reviews but here are some things I should change:

- rename $myTerm to $term
we don't need to specify "my" here, it's alread *my* terminal of choice. no need to make the variable longer than it needs to be
- do not define mediakeys by default, but instead add a commented out section
it really can differ per device, lets not make assumptions
- change ***"bothersome*** yellow warning 
it sounds like it was written by someone butthurt by said option, just say it removes the warning message
- why the fuck do we have the large ascii art on top
seriously that's a terrible idea, you're adding more scroll space and a few bytes to the filesize. absolutely no benefits whatsoever

- *maybe* add a second term variable that activates on a different keybind (i.e mod + shift + return) so people don't think that's a "default", but rather a suggestion

**Comment by vaxerski** [comment]
Agreed with all points presented by NotAShelf

**Comment by System-x64** [comment]
now?

File: example/hyprland.conf [hunk 1]
```diff
@@ -1,17 +1,10 @@
-# ██╗  ██╗██╗   ██╗██████╗ ██████╗ ██╗      █████╗ ███╗   ██╗██████╗     
-# ██║  ██║╚██╗ ██╔╝██╔══██╗██╔══██╗██║     ██╔══██╗████╗  ██║██╔══██╗    
-# ███████║ ╚████╔╝ ██████╔╝██████╔╝██║     ███████║██╔██╗ ██║██║  ██║    
-# ██╔══██║  ╚██╔╝  ██╔═══╝ ██╔══██╗██║     ██╔══██║██║╚██╗██║██║  ██║    
-# ██║  ██║   ██║   ██║     ██║  ██║███████╗██║  ██║██║ ╚████║██████╔╝    
-# ╚═╝  ╚═╝   ╚═╝   ╚═╝     ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝╚═╝  ╚═══╝╚═════╝     
-                                                                       
-# This is an example Hyprland config file.
+#Example hyprland.conf. Copy this at ~/.config/hypr/hyprland.conf                                                              # This is an example Hyprland config file.
```
Comment (NotAShelf) [inline example/hyprland.conf:1]: Copy this *to* `~/.config/hypr/hyprland.conf`. Not "at"

File: src/config/defaultConfig.hpp [hunk 2]
```diff
@@ -3,20 +3,13 @@
...
-                                                                       
-# This is an autogenerated Hyprland config file.
+# Autogenerated by Hyprland.                                                                  # This is an autogenerated Hyprland config file.
 # Refer to the wiki (https://wiki.hyprland.org/) for more information.
 
 #! Please note, not all available settings / options are set here.
 #! For a full list, see the wiki (https://wiki.hyprland.org/)
 
-###! REMOVE THIS LINE TO DISABLE THAT BOTHERSOME YELLOW WARNING!
+###! REMOVE THIS LINE TO REMOVE THAT YELLOW WARNING!
```
Comment (NotAShelf) [inline src/config/defaultConfig.hpp:12]: "That yellow warning"? What yellow warning?
Make sure to clearly specify what marking we're talking about. For example
> "Remove this line to disable the yellow warning on top"

Comment (NotAShelf) [reply src/config/defaultConfig.hpp:12]: Noticed this is at the wrong place, but you get the point.

File: example/hyprland.conf [hunk 3]
```diff
@@ -190,17 +184,17 @@ bindm = $mainMod, mouse:272, movewindow
 bindm = $mainMod, mouse:273, resizewindow
 
 # Media Control keys
-#! Dependencies: pulseaudio-ctl brightnessctl playerctl
+# Dependencies: pulseaudio-ctl brightnessctl playerctl
```
Comment (NotAShelf) [inline example/hyprland.conf:187]: Maybe also specify those are *optional* dependencies, for if people desire to use it.
Even better, I would remove this section entirely and put it under a tips and tricks page on the wiki.

**Comment by vaxerski** [comment]
image

sir for fucks sake

**Comment by System-x64** [comment]
> image
> 
> sir for fucks sake

6A7D40E5-A585-4878-8CDE-AC9316DEAD4C I have write access enabled from the start

**Comment by System-x64** [comment]
Nvm it’s too much work for just one small file, all these commits made it a mess anyways, i’ll better close it

**Pull request closed without merging**